ocs_ci/helpers/dr_helpers.py

```
@@ -2920,3 +2923,110 @@ def validate_protection_label(kind, namespace, protection_name=None):
+
+    if action == "gather":
+        try:
+            logger.info(
+                f"Running ODF DR gather application for DRPC '{drpc_name}' "
+                f"in namespace '{namespace}'"
+            )
+            result = odf_cli_runner.run_command(cmd_args)
+            stdout = result.stdout.decode()
+            logger.info(f"ODF DR gather application output:\n{stdout}")
+            return stdout
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)